### PR TITLE
Rubocop rspec fixes

### PR DESCRIPTION
**Pull request by smeny**

### CODE REVIEW

File: spec/features/form/saving_data_spec.rb [hunk 1]
```diff
@@ -88,11 +88,11 @@
     # Something about our styling makes the selenium webdriver think the actual radio buttons are not visible so we pass false here
```
Comment (baarkerlounger) [inline spec/features/form/saving_data_spec.rb]: Looks like this comment can probably go now?

File: spec/models/case_log_spec.rb [hunk 2]
```diff
@@ -764,166 +787,113 @@
       end
     end
 
-    context "Validate pregnancy questions" do
```
Comment (baarkerlounger) [inline spec/models/case_log_spec.rb:767]: Are these being deleted or just moved?

File: spec/models/validations/local_authority_validations_spec.rb [hunk 3]
```diff
@@ -1,55 +1,34 @@
...
-RSpec.describe CaseLog do
-  let(:owning_organisation) { FactoryBot.create(:organisation) }
-  let(:managing_organisation) { owning_organisation }
-
-  before do
-    RequestHelper.stub_http_requests
-  end
-
-  describe "#new" do
-    it "raises an error when previous_postcode is present and invalid" do
```
Comment (baarkerlounger) [inline spec/models/validations/local_authority_validations_spec.rb:13]: This moves into the model spec - nice!

**Review by baarkerlounger — APPROVED** [review]
Couple of comments/questions but in general this might be my favourite ever PR. :trophy:

File: spec/models/form_handler_spec.rb [hunk 4]
```diff
@@ -2,26 +2,27 @@
...
+      form_handler = described_class.instance
       form = form_handler.get_form(test_form_name)
       expect(form).to be_a(Form)
       expect(form.pages.count).to eq(29)
     end
   end
 
-  it "should only load the form once at boot time" do
-    form_handler = FormHandler.instance
+  it "onlies load the form once at boot time" do
```
Comment (smeny) [inline spec/models/form_handler_spec.rb]: typo here

File: spec/models/case_log_spec.rb (hunk 2 above)
Comment (smeny) [reply spec/models/case_log_spec.rb:767]: The whole section was duplicated, it was around line ~100-120

File: spec/features/form/saving_data_spec.rb (hunk 1 above)
Comment (smeny) [reply spec/features/form/saving_data_spec.rb]: done

**Pull request merged**